app/src/components/CopiableCode.tsx

```diff
@@ -0,0 +1,43 @@
+import { HStack, Icon, IconButton, Tooltip, Text } from "@chakra-ui/react";
+import { useCallback, useState } from "react";
+import { MdContentCopy } from "react-icons/md";
+
+const CopiableCode = ({ code }: { code: string }) => {
+  const [copied, setCopied] = useState(false);
+
+  const copyToClipboard = useCallback(() => {
+    const onCopy = async () => {
+      console.log("copied!");
+      await navigator.clipboard.writeText(code);
+      setCopied(true);
+    };
+    void onCopy();
```

Comment:
what is the `void` identifier for? I'm not familiar with that pattern.

Reply:
It allows us to avoid awaiting an asynchronous function without any eslint errors. I don't think it used to be necessary, so I must have updated my linter recently.

Reply:
I actually like the fact that eslint tells us if we forget to report an unhandled promise exception -- keeps us from silently swallowing errors accidentally.